examples/platform_services/README.md

```diff
@@ -6,61 +6,9 @@ You can read more about
 [accessing platform and third-party services in Flutter](https://flutter.io/platform-services/).
 
 ## iOS
-
-### Configure
-
-Create an `ios/Flutter/Generated.xcconfig` file with this entry:
-
- * `FLUTTER_ROOT=[absolute path to the Flutter SDK]`
-
-There are a number of other parameters you can control with this file:
-
- * `FLUTTER_APPLICATION_PATH`: The path to the directory that contains your
-   `pubspec.yaml` file relative to your `xcodeproj` file.
- * `FLUTTER_BUILD_MODE`: Whether to build for `debug`, `profile`, or `release`.
-   Defaults to `release`.
- * `FLUTTER_TARGET`: The path to your `main.dart` relative to your
-   `pubspec.yaml`. Defaults to `lib/main.dart`.
- * `FLUTTER_FRAMEWORK_DIR`: The absolute path to the directory that contains
-   `Flutter.framework`. Defaults to the `ios-release` version of
-   `Flutter.framework` in the `bin/cache` directory of the Flutter SDK.
-
-### Build
-
-Once you've configured your project, you can open `ios/Runner.xcodeproj`
-in Xcode and build the project as usual.
+You can open `ios/Runner.xcworkspace` in Xcode and build the project as usual.
```

Comment:
I think we should add both. Since developers might choose to use this as a starting point, it is nice to know about both options.